invenio_github/translations/it/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,60 @@
+# Translations template for invenio-github.
+# Copyright (C) 2016 CERN
+# This file is distributed under the same license as the invenio-github
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: invenio-github 1.0.0a4.dev20160804\n"
+"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
+"POT-Creation-Date: 2016-08-25 10:50+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Alizee Pace <alizee.pace@gmail.com>, 2016\n"
+"Language-Team: Italian (https://www.transifex.com/inveniosoftware/teams/23537/it/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+"Language: it\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: invenio_github/models.py:48
+msgid "Received"
+msgstr "Ricevuto"
+
+#: invenio_github/models.py:49
+msgid "Processing"
+msgstr "In trattamento"
+
+#: invenio_github/models.py:50
+msgid "Published"
+msgstr "Pubblicato"
+
+#: invenio_github/models.py:51
+msgid "Failed"
+msgstr "Fallito"
+
+#: invenio_github/templates/invenio_github/settings/helpers.html:78
+msgid ""
+"This repository is managed by another user of your GitHub organization."
+msgstr ""
+"Questo repository è gestito da un'altro utente della tua organizzazione "
+"GitHub."
+
+#: invenio_github/templates/invenio_github/settings/index_item.html:26
+msgid "Go to GitHub."
+msgstr "Vai su GitHub."
+
+#: invenio_github/views/github.py:89
+msgid "<i class=\"fa fa-github fa-fw\"></i> GitHub"
```

Comment:
Can you change the position of `_` in that string, as it does not make sense to offer HTML code snippets for translations?  The text to translate would be only "GitHub". However, the name GitHub can probably stay like that in all the languages, so this phrase could be possibly removed altogether...